src/videojs-http-streaming.js

```diff
@@ -682,6 +698,7 @@ class VhsHandler extends Component {
       'experimentalBufferBasedABR',
       'liveRangeSafeTimeDelta',
       'experimentalLLHLS',
+      'useNetworkInformation',
```

Comment:
I think `useNetworkInformationApi`, although a bit more verbose, directly points to the API we're referencing.

We should also add an entry to the README.

Reply:
SGTM